### PR TITLE
Dev kmr

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: .github/actions/get-repo-name/action.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  repo-name:
+    description: "The repository name for the running workflow"
+    value: ${{ steps.get-repo-name.outputs.repo-name }}
+runs:
+  using: "composite"
+  steps:
+    - name: "get repo name"
+      id: get-repo-name
+      shell: bash
+      run: ${{ github.action_path }}/get-repo-name.sh ${{ github.repository }}
```
Comment (rafaelfolco) [inline .github/actions/get-repo-name/action.yml:13]: wondering why you can't simply use github.repository

File: README.md [hunk 2]
```diff
@@ -12,75 +12,130 @@ The intent/goal here is that the upstream code in the primary branch for all rep
...
+
+#### Runnable Workflows
+
+These workflows are what run as part of pull requests that are submitted against the Crucible-CI subproject itself.
+
+##### faux-crucible-ci
+
+The [faux-crucible-ci](.github/workflows/faux-crucible-ci.yaml) workflow is a "fake" workflow that is run when the pull request in question does not actually affect runtime behavior (such as documentation only changes).  It provides "null" jobs that satisfy Crucible-CI's GitHub branch protection rules.
+
+##### run-core-crucible-ci
```
Comment (rafaelfolco) [inline README.md:127]: why do we need core-crucible-ci and run-core ?

**Review by rafaelfolco — APPROVED** [review]
lgtm

File: .github/actions/get-repo-name/action.yml (hunk 1 above)
Comment (k-rister) [reply .github/actions/get-repo-name/action.yml:13]: The `github.repository` context variable is of the form `<project>/<repo>` or `<owner>/<repo>` and several of the places that we are wanting to use this information are either a URL or a file path so the presence of the `/` in the string is destructive.  Sometimes we may want to use the repo name in a workflow itself so it is difficult (or at least more difficult than it should be) to do the string parsing natively.  Writing a simple action just makes an easy way to access this information in the format we want directly in a workflow if needed.

File: README.md (hunk 2 above)
Comment (k-rister) [reply README.md:127]: The reusable workflow (`core-crucible-ci.yaml`) does not execute on it's own, it must be called from something to get it to execute.  The `run-core-crucible-ci.yaml` workflow is that caller and is a very simple example of how other subprojects will be calling the reusable workflows.

**Pull request merged**